{"name": "测试数据：【 1532341，111111a，{except_result}，10000，token】", "status": "broken", "statusDetails": {"message": "TypeError: post() got multiple values for argument 'url'", "trace": "self = <testcases.api_test.test_03_login.TestUserLogin object at 0x0000022AA5840B70>\nusername = '1532341', password = '111111a', login_type = 1, except_code = 10000\nexcept_msg = 'token'\nheaders = {'Content-Type': 'application/json', 'select_nation_id': 'RU'}\nurl = '/idserverhq/public/auth/login'\n\n    @allure.story(\"用例--登录用户\")\n    @allure.description(\"该用例是针对获取用户登录接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{password}，{except_result}，{except_code}，{except_msg}】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, password, login_type, except_code, except_msg, headers, url\",\n                             api_data[\"test_login_user\"])\n    \n    def test_login_user(self, username, password, login_type, except_code, except_msg,headers,url):\n        logger.info(\"*************** 开始执行用例 ***************\")\n>       result = login_user(username, password, login_type,headers,url)\n\ntest_03_login.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\operation\\user.py:103: in login_user\n    res = user.login(json=payload, headers=header, url=url)\n..\\..\\api\\user.py:27: in login\n    return self.post(**kwargs)\n..\\..\\core\\rest_client.py:31: in post\n    return self.request(\"POST\", data, json, **kwargs)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <api.user.User object at 0x0000022AA4F88320>, method = 'POST'\ndata = None\njson = {'login_type': 1, 'password': '111111a', 'username': '1532341'}\nkwargs = {'headers': {'Content-Type': 'application/json', 'select_nation_id': 'RU'}, 'url': '/idserverhq/public/auth/login'}\nheaders = {'Content-Type': 'application/json', 'select_nation_id': 'RU'}\nparams = None, files = None, cookies = None\nurl = 'https://dev2-hq-kong.tiens.com/idserverhq/public/auth/login'\n\n    def request(self, method, data=None, json=None, **kwargs):\n        # url = self.api_root_url + url\n        headers = dict(**kwargs).get(\"headers\")\n        params = dict(**kwargs).get(\"params\")\n        files = dict(**kwargs).get(\"params\")\n        cookies = dict(**kwargs).get(\"params\")\n        url = dict(**kwargs).get(\"url\")\n        if url == \"/idserverhq/public/auth/login\":\n            url = self.api_root_login_url + url\n        else:\n            url = self.api_root_url + url\n        self.request_log(url, method, data, json, params, headers, files, cookies)\n        if method == \"GET\":\n            return self.session.get(url, **kwargs)\n        if method == \"POST\":\n>           return self.session.post(url, data,json, **kwargs)\nE           TypeError: post() got multiple values for argument 'url'\n\n..\\..\\core\\rest_client.py:62: TypeError"}, "description": "该用例是针对获取用户登录接口的测试", "attachments": [{"name": "log", "source": "23d96225-6f1d-4500-afc4-a24afea7ef1b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'1532341'"}, {"name": "password", "value": "'111111a'"}, {"name": "login_type", "value": "1"}, {"name": "except_code", "value": "10000"}, {"name": "except_msg", "value": "'token'"}, {"name": "headers", "value": "{'Content-Type': 'application/json', 'select_nation_id': 'RU'}"}, {"name": "url", "value": "'/idserverhq/public/auth/login'"}], "start": 1669426779858, "stop": 1669426779858, "uuid": "f59d8d85-c71b-4b6a-a757-d732dd8ea5d5", "historyId": "0de4ecfae382e86437822b0a68f446dd", "testCaseId": "7272db244dabef390096479ae3196642", "fullName": "testcases.api_test.test_03_login.TestUserLogin#test_login_user", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "story", "value": "用例--登录用户"}, {"name": "feature", "value": "用户登录模块"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_03_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "lijuan"}, {"name": "thread", "value": "12340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_03_login"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}